sdk/digitaltwins/Azure.DigitalTwins.Core/src/DigitalTwinsClientOptions.cs

```diff
@@ -12,7 +12,7 @@ namespace Azure.DigitalTwins.Core
     /// </summary>
     public class DigitalTwinsClientOptions : ClientOptions
     {
-        internal const ServiceVersion LatestVersion = ServiceVersion.V2022_05_31;
+        internal const ServiceVersion LatestVersion = ServiceVersion.V2023_02_27_Preview;
```

Comment:
Is it expected for the APIs in main/master to be referencing a Preview version of the service?

Reply:
I guess same response as Santhosh here - @sjiherzig any insights on a separate branch for preview apis?

Reply:
This PR will be updated with the GA version when that is available in the swagger

Reply:
You cannot take an SDK to GA if it references a preview version. We ran into this in the past. We'll need to add a new stable API version in this case, and reference that in the SDKs.